package.json

```diff
@@ -102,31 +102,18 @@
     "typescript-eslint": "^7.9.0"
   },
   "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
+    "moduleFileExtensions": ["js", "json", "ts"],
     "rootDir": ".",
-    "setupFiles": [
-      "dotenv/config"
-    ],
+    "setupFiles": ["./jest-setup.ts"],
```

Comment:
Most of this is formatting, but here is where it switches to use the jest-setup file instead of just dotenv directly